src/Core/src/Hosting/MauiApp.cs

```diff
@@ -1,47 +1,27 @@
 ﻿using System;
-using System.Threading;
-using System.Threading.Tasks;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Hosting;
-using Microsoft.Extensions.Logging;
 
 namespace Microsoft.Maui.Hosting
 {
-	public sealed class MauiApp : IHost
+	public sealed class MauiApp : IDisposable
 	{
-		private readonly IHost _host;
+		private readonly IServiceProvider _services;
 
-		internal MauiApp(IHost host)
+		internal MauiApp(IServiceProvider services)
 		{
-			_host = host;
-			Logger = host.Services.GetRequiredService<ILoggerFactory>().CreateLogger(Environment.ApplicationName);
+			_services = services;
 		}
 
 		/// <summary>
 		/// The application's configured services.
 		/// </summary>
-		public IServiceProvider Services => _host.Services;
+		public IServiceProvider Services => _services;
 
 		/// <summary>
 		/// The application's configured <see cref="IConfiguration"/>.
 		/// </summary>
-		public IConfiguration Configuration => _host.Services.GetRequiredService<IConfiguration>();
-
-		/// <summary>
-		/// The application's configured <see cref="IHostEnvironment"/>.
-		/// </summary>
-		public IHostEnvironment Environment => _host.Services.GetRequiredService<IHostEnvironment>();
-
-		/// <summary>
-		/// Allows consumers to be notified of application lifetime events.
-		/// </summary>
-		public IHostApplicationLifetime Lifetime => _host.Services.GetRequiredService<IHostApplicationLifetime>();
-
-		/// <summary>
-		/// The default logger for the application.
-		/// </summary>
-		public ILogger Logger { get; }
+		public IConfiguration Configuration => _services.GetRequiredService<IConfiguration>();
 
 		/// <summary>
 		/// Initializes a new instance of the <see cref="MauiApp"/> class with preconfigured defaults.
```

Comment:
@PureWeen - do you see a need / use case for the `public static MauiApp Create()` method?

I don't think anyone can actually use this method because it won't register an `IApplication` service without `.UseMauiApp<App>` being called:

```
System.InvalidOperationException: No service for type 'Microsoft.Maui.IApplication' has been registered.
```

https://github.com/dotnet/maui/blob/3041455afbe1e2db73883cf7c0442ce765e38faf/src/Core/src/Platform/Android/MauiApplication.cs#L37

And once the `MauiApp` object has been built, you can't add more services.

I think this API is just not usable in its current state. Thoughts on removing it as well?

Reply:
FYI - my latest change removed this method as well. I can back that out if you think there is value in this method. But I don't see it.

Reply:
That seems fine

@Eilon ? It looks like you added `Create` as part of this PR

https://github.com/dotnet/maui/commit/53a1d32d9114893a4a92dec727e6a2e22b6305e2#diff-402d30e3cca775dabd446e3058ee424391f17edc32f27ef748d9846b0cd78eb9R45

Reply:
Yeah it doesn't look needed. It was either my own error while iterating, or perhaps it was copied from what ASP.NET Core does (which maybe had something for some historical reason). Either way, not needed.